src/api/app/assets/stylesheets/webui/bootstrap_variables/colors.scss

```diff
@@ -1,5 +1,4 @@
 // Override Bootstrap's default colors
-$body-bg: #f6f6f6;
```

Comment:
`$body-bg` is still used here: https://github.com/openSUSE/open-build-service/blob/55a05a719d9db402c32f4b69f136afbe5cd7ad3e/src/api/app/assets/stylesheets/webui/package-attributes.scss#L7

Should we remove, or at least adapt, that line?

Reply:
Taking in account that:
1. the variable `$body-bg` is still being used in the `package-attributtes.scss` for setting the `background-color`, and
2. removing the line `$body-bg: #f6f6f6;` prevents from overriding the value of the variable `$body-bg`, so it would take the default value,

I expect that background to change its colour from `'#f6f6f6'` to the default value.

Should this PR change that background color to the default from Bootstrap? Is this change intended?